Scripts/lombiq-asynceditorwrapper.js

```diff
@@ -47,19 +47,19 @@
             var $editorPlaceholder = plugin.element.find(plugin.settings.editorPlaceholderElementClass);
 
             if (plugin.settings.addNewItemActionElementClass) {
-                $(document).find(plugin.settings.addNewItemActionElementClass).on("click", function () {
+                $(document).on("click", plugin.settings.addNewItemActionElementClass, function () {
                     plugin.loadEditor($(this).attr("data-contentItemId"), $editorPlaceholder);
                 });
             }
 
             if (plugin.settings.editItemActionElementClass) {
-                plugin.element.find(plugin.settings.editItemActionElementClass).on("click", function () {
+                $(document).on("click", plugin.settings.editItemActionElementClass, function () {
                     plugin.loadEditor($(this).attr("data-contentItemId"), $editorPlaceholder);
                 });
             }
 
             if (plugin.settings.deleteItemActionElementClass) {
-                plugin.element.find(plugin.settings.deleteItemActionElementClass).on("click", function () {
+                $(document).on("click", plugin.settings.deleteItemActionElementClass, function () {

```

Comment:
So, this won't throw an error if the target element do not exist in the DOM, is that correct?

Reply:
https://learn.jquery.com/events/event-delegation/